src/inc/quic_datapath.h

```diff
@@ -379,6 +379,19 @@ typedef struct CXPLAT_TCP_DATAPATH_CALLBACKS {
 
 } CXPLAT_TCP_DATAPATH_CALLBACKS;
 
+typedef struct CXPLAT_WORKER CXPLAT_WORKER;
```

Comment:
Maybe move this stuff to quic_platform.h, after/around `CXPLAT_EXECUTION_CONTEXT`?